<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CriticalPointsPanel</class>
 <widget class="QWidget" name="CriticalPointsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CriticalPointsPanel</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="methodComboBox">
        <item>
         <property name="text">
          <string>Geodesic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z-Value</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Point-Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Curvature</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quadratic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Smoothing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="smoothingSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="smoothingSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="smoothingTypeCheckBox">
        <property name="text">
         <string>Smoothing on Actual Mesh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
